crypto/rsa/rsa_lib.c

```diff
@@ -259,6 +327,37 @@ void RSA_get0_factors(const RSA *r, const BIGNUM **p, const BIGNUM **q)
         *q = r->q;
 }
 
+size_t RSA_get_multi_prime_extra_count(const RSA *r)
+{
+    int pnum;
+
+    pnum = sk_RSA_PRIME_INFO_num(r->prime_infos);
+    if (pnum <= 0)
+        pnum = 0;
+    return (size_t)pnum;
+}
+
+int RSA_get0_multi_prime_factors(const RSA *r, const BIGNUM **oth_primes)
```

Comment:
How about declaring last argument as `const BIGNUM *oth_primes[]`?

Reply:
This remark applies to multiple cases.

Reply:
what's the benefit by changing to this?

Reply:
I for one view it as more educational and therefore transparent. Indeed, how does caller view it? Rather specifically as array of pointers to BIGNUMs. Even this code uses something[RSA_MAX_PRIME_NUM] at multiple occasions.

Reply:
Addressed